domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -24,7 +24,7 @@ class StoryProgressTestHelper @Inject constructor(
    * Creates a partial story progress for a particular profile.
    *
    * @param profileId the profile we are setting partial progress of the fraction story for
-   * @param timestampOlderThanOneWeek if the timestamp for this topic progress is more than one week ago
```

Comment:
Actually, can we go the other way & update the parameter to match this name, instead? I think 'one week' is clearer than 'a week'.

Reply:
Sure thing, first thing I do when I hop onto my workstation.

Reply:
The PR reviewer can resolve the conversation after confirming the suggested changes are done or if anything left to discuss. The conversation should not be resolved by PR authors.

Reply:
Resolved!

Reply:
Sorry my bad.

Reply:
No worries. :) Thanks for fixing this @OBITORASU!